packages/accounts-password/password_server.js

```diff
@@ -169,18 +169,29 @@ Accounts.registerLoginHandler("password", options => {
 
   check(options, {
     user: Accounts._userQueryValidator,
-    password: passwordValidator
+    password: passwordValidator,
+    token: Match.Optional(NonEmptyString),
   });
 
 
   const user = Accounts._findUserByQuery(options.user, {fields: {
     services: 1,
+    twoFactorAuthentication: 1,
     ...Accounts._checkPasswordUserFields,
   }});
   if (!user) {
     Accounts._handleError("User not found");
   }
 
+  if(Accounts.checkUserHas2FAEnabled && Accounts.checkUserHas2FAEnabled(options.user)){
```

Comment:
New lines should have Prettier applied. But pay attention, just new lines, not the entire file.

Reply:
When this `Accounts.checkUserHas2FAEnabled` is going to be falsy?

Reply:
It's falsy when the package accounts-2fa is not present